src/command/restore/restore.c

```diff
@@ -1367,6 +1367,8 @@ restoreRecoveryOption(unsigned int pgVersion)
             KeyValue *optionReplace = kvNew();
 
             kvPut(optionReplace, VARSTR(CFGOPT_EXEC_ID_STR), NULL);
+            kvPut(optionReplace, VARSTR(CFGOPT_JOB_RETRY_STR), NULL);
+            kvPut(optionReplace, VARSTR(CFGOPT_JOB_RETRY_INTERVAL_STR), NULL);
```

Comment:
This is only to prevent it being passed to archive-get if it was passed to restore on the command line.

If it is set in pgbackrest.conf then restore and archive-get will both use the same settings, or they can be customized per command.